resources/sdk/clisdkclient/templates/restclient.mustache

```diff
@@ -129,7 +138,34 @@ func (r *RESTClient) callAPI(method string, uri string, data string) (string, er
 			}
 		}
 	}
+    
+    if r.configuration != nil {
+		proxyConfiguration := r.configuration.ProxyConfiguration()
+	if proxyConfiguration != nil && proxyConfiguration.Protocol != "" {
+		var proxyUrl *url.URL
+
+		if proxyConfiguration.Auth != nil && proxyConfiguration.Auth.UserName != "" && proxyConfiguration.Auth.Password != "" {
+			proxyUrl = &url.URL {
+      		Scheme: proxyConfiguration.Protocol,
+     	    User:   url.UserPassword(proxyConfiguration.Auth.UserName, 
+	  							proxyConfiguration.Auth.Password),
+      		Host:   proxyConfiguration.Host + ":" + proxyConfiguration.Port,
+    		}
+		} else {
+				urlString := proxyConfiguration.Protocol + "://" +
+						proxyConfiguration.Host  + ":" +
+						proxyConfiguration.Port
+				proxyUrl, _ = url.Parse(urlString)
+		}
```

Comment:
The formatting here is off. Sometimes manually tabbing out the lines in vs code can fix the issue. I would also run `go fmt` on the `.go` files if you haven't already. I think maybe there are some formatting issues that are stopping github from highlighting the changes effectively.